src/components/Item.js

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+import { updateDoc, serverTimestamp, doc } from 'firebase/firestore';
+import { db } from '../lib/firebase';
+import './Item.css';
+
+function Item({ item, userToken }) {
+  const [checked, setChecked] = useState(false);
+  const [daysSincePurchased, setDaysSincePurchased] = useState();
+
+  useEffect(() => {
+    if (item.lastPurchased) {
+      const lastPurchasedSeconds = item.lastPurchased.seconds;
+      const dateNowSeconds = Date.now() / 1000;
+      const differenceInSeconds = dateNowSeconds - lastPurchasedSeconds;
+
+      setDaysSincePurchased(differenceInSeconds / 86400);
+    }
+  }, [item]);
+
+  useEffect(() => {
+    if (daysSincePurchased > 1) {
```

Comment:
You can combine this if with the one on line 29

```suggestion
if (daysSincePurhcased > 1 || daysSincePurchase === null)
```